CONTRIBUTING.md

```diff
@@ -95,7 +97,7 @@ The changelog should follow the following convention:
 
 | Version | Description |
 |---------|-------------|
-| 0.1.0 | [PR#49](https://github.com/bbc/psammead/pull/49) Initial creation of package |
+| 1.0.0 | [PR#49](https://github.com/bbc/psammead/pull/49) Initial creation of package |
```

Comment:
Version 1.0.0 should be used once we've completed an accessibility swarm with our supported assistive tech and addressed changes necessary. https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology
Likewise have checked browser compatibility against our support table. https://github.com/bbc/psammead/blob/latest/README.md#bar_chart-support-levels